frontend/components/generic/LogoBase64.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box, SxProps, Theme } from '@mui/material';
+
+interface Props {
+  // Base64 encoding of an image (prefix should not be included).
+  base64Src: string;
```

Comment:
```suggestion
  base64Src: Base64String;
```

To update after my PR is merged.

Reply:
not relevant anymore since we're using <img>